modules/src/ics02_client/state.rs

```diff
@@ -43,6 +43,17 @@ pub trait ClientState: Clone + std::fmt::Debug {
 
     /// Verification functions as specified in:
     /// https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics
+    ///
+    fn verify_client_full_state(
```

Comment:
The other two `verify_*` functions here appear in [ICS02](https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#required-functions) but this `verify_client_full_state ` one does not ring a bell. All implementations of `verify_client_full_state` are also `unimplemented` so it's not really clear what the function does by just looking at the code. Can we have a comment here (describing the purpose of the function like we do for `verify_client_consensus_state` or `verify_connection_state`)?